docs/reference/enable-gpu.rst

```diff
@@ -1,46 +1,153 @@
+
+Ensure DaemonSet is Ready
+"""""""""""""""""""""""""
+
+First we need to ensure that the DaemonSet for the Operator Validator is
+created:
+
+
+.. code-block:: bash
+
+  while ! sudo microk8s.kubectl get ds -n gpu-operator-resources nvidia-operator-validator
+  do
+    sleep 5
+  done
+
+.. note::
+   It will take some seconds for the DaemonSet to get created. The above command will
+   return a message like the following at the beginning:
+   ``Error from server (NotFound): daemonsets.apps "nvidia-operator-validator" not found``
+
+The expected output should be:
+
+.. code-block:: text
+
+   Error from server (NotFound): daemonsets.apps "nvidia-operator-validator" not found
+   ...
+   Error from server (NotFound): daemonsets.apps "nvidia-operator-validator" not found
+   NAME                        DESIRED   CURRENT   READY   UP-TO-DATE   AVAILABLE   NODE SELECTOR                                   AGE
+   nvidia-operator-validator   1         1         0       1            0           nvidia.com/gpu.deploy.operator-validator=true   17s
+
+Ensure the Validator succeeded
+""""""""""""""""""""""""""""""
+
+Next we'll need to wait for the Validator Pod to succeed:
+
+.. code-block:: bash
+
+  echo "Waiting for the NVIDIA Operator validations to complete..."
+  while ! sudo microk8s.kubectl logs -n gpu-operator-resources -l app=nvidia-operator-validator -c nvidia-operator-validator | grep "all validations are successful"
+  do
+    sleep 5
+  done
+
+.. note::
+   It will take some seconds for the Pod to get initialized . The above command will
+   return a message like the following at the beginning:
+   ``Error from server (BadRequest): container "nvidia-operator-validator" in pod "nvidia-operator-validator-4rq5n" is waiting to start: PodInitializing``
+
+The expected output should be:
+
+.. code-block:: text
+
+   Error from server (BadRequest): container "nvidia-operator-validator" in pod "nvidia-operator-validator-4rq5n" is waiting to start: PodInitializing
+   ...
+   Error from server (BadRequest): container "nvidia-operator-validator" in pod "nvidia-operator-validator-4rq5n" is waiting to start: PodInitializing
+   all validations are successful
+
+Verify the DSS detects the GPU
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. code-block:: bash
+
+  dss status
+
+Expected output:
+
+.. code-block:: bash
+
+  [INFO] MLflow deployment: Ready
+  [INFO] MLflow URL: http://10.152.183.74:5000
+  [INFO] GPU acceleration: Enabled (NVIDIA-GeForce-RTX-3070-Ti)
 
-* NVIDIA drivers installed on your machine.
-* The MicroK8s runtime configured to be able to use the Nvidia drivers.
+.. note::
 
-The `MicroK8s GPU add-on`_ is installing the NVIDIA Operator. The NVIDIA operator can either install or detect the NVIDIA drivers if they are already present on the host machine.
+  The GPU model `NVIDIA-GeForce-RTX-3070-Ti` will be different depending on your device.
 
-Before installing DSS:
+Launch GPU-enabled Notebook
```

Comment:
```suggestion
Launch GPU-enabled notebook
```
Do we consider notebook an specific product or a general term? Not sure here. The latter would be lower case.

Reply:
Pushed a commit 6d795e7